runtime/libffi/z/ffi64.c

```diff
@@ -977,6 +1023,7 @@ ffi_call(ffi_cif *cif,
   switch (cif->abi)
     {
       case FFI_SYSV:
+        {
```

Comment:
The addition of this line and line 1035 is not necessary: please remove them.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/38245ec2872c2b2664f6fdd3947f80e71e63c57d#diff-7e388e6ac3235f9470ab3d7607ffbe1c8fceacddd69ac992fdae6cda3e99257d